python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/search.py

```diff
@@ -114,6 +112,18 @@ def search(
             total_results=response["total"], page_results=page_results, facets=facets
         )
 
+    @staticmethod
+    def _get_matched_fields(result: dict) -> dict:
+        fields = result.get("matchedFields", [])
+        matched_fields = {}
+        for field in fields:
+            name = field.get("name")
+            value = field.get("value")
+            if name == "customProperties" and value != "":
```

Comment:
Can you add a test for this case? I think the rest of this function is tests already but not the special case for customProperties

Reply:
Updated the query test case with a custom property match field.